### ISSUE TITLE
Which fork should I pick?

### ISSUE THREAD

**Issue by sirasjad** [havardgulldahl/jottalib#149]
There are so many different forks, which one should I use? The official repo by havardgulldahl hasn't been updated for over a year, while Fjodor42's repo was updated 20 hours ago. Is his repo stable? Thanks in advance.

**Comment by Fjodor42** [comment]
Hi @sirajuddin97,

You are more than welcome to clone from mine, but, obviously, I would be wary of giving any guaranties in any legal sense.

What I will say, though, is that @havardgulldahl|s code seemed to work fine for me, with the caveat that the 'requests' library had to be below a certain version, which was the first change I attempted (and with which I seemed to succeed).

I did note some odd behaviour here and there, though, so the 'overhaul' PR reflects a larger number of changes that I made, although quite a few commits merely exist to try to get his tests (thank you for them, @havardgulldahl) pass under the AppVeyor build service's Windows environment, and later also the Python 3 tests (until I found out how to run them locally first).

My primary (only?) use case is backing up to Jotta via duplicity, which most certainly seems to work for me, but I may also have changed the syntax for the cli tools ('m not entirely sure), without actually updating any documentation.

That said, I would welcome feedback / bug reports, and try to update as necessary and as time permits, and the same goes for functionality that has yet to be implemented, but I would really like to hear from @havardgulldahl as to whether he wants to get back into this, or if he would prefer for me to do a fork in the "real" sense that it wouldn't refer back to his repo any more.

Until such a confirmation (or until a reasonable amount of time has passed), I shall attempt maintain my fork under the current arrangement, but will ask that people ping me if bug reports are opened or need attention here in this repo.

tl;dr; I stand by my fork in the "works for me" sense, and will try to work out forthcoming issues, but I will not do a "hard fork" until given permission from @havardgulldahl, or until enough time has passed that it can be safely assumed that "silence implies consent".

**Comment by sirasjad** [comment]
I'm willing to help you continue this project, @Fjodor42, as I have been focusing a lot on Python development lately. I have tried to contact @havardgulldahl as well, sent him an email but I can't seem to get in touch with him. I don't think he's coming back anytime soon.

My suggesion is that you copy all your files to a brand new official repo (call it _jottalib v2_ or something), give @havardgulldahl credits for his awesome work and then continue the development from there. I believe that's the only way. I can help you re-write the documentation and solve issues, correct bugs, etc.
I hope we can make this work!

**Comment by Fjodor42** [comment]
@sirajuddin97: You are now invited on as a collaborator on https://github.com/Fjodor42/jottalib-ng

You, and everyone else, are welcome to look at the first few issues I created, and open new ones

**Comment by sirasjad** [comment]
Let's get the party started then!
